src/hub.html

```diff
@@ -156,12 +156,37 @@
 
                         <template data-name="Spine">
                             <a-entity is-remote-hover-target personal-space-invader="radius: 0.2; useMaterial: true;" bone-visibility>
-                                <a-entity billboard matrix-auto-update visibility-while-frozen="withinDistance: 10; requireHoverOnNonMobile: false;">
-                                    <a-entity is-remote-hover-target tags="singleActionButton:true;" mixin="rounded-text-button" block-button ui-class-while-frozen position="0 0 .35">
-                                        <a-entity text="value:Hide; width:2.5; align:center;" position="0 0 0.01"></a-entity>
-                                    </a-entity>
-                                    <a-entity is-remote-hover-target tags="singleActionButton:true;" mixin="rounded-text-button" visible-if-permitted="kick_users" kick-button ui-class-while-frozen position="0 -0.25 .35">
-                                        <a-entity text="value:Kick; width:2.5; align:center;"  visible-if-permitted="kick_users" position="0 0 0.01"></a-entity>
+                                <a-entity class="ui interactable-ui">
+                                    <a-entity matrix-auto-update billboard class="freeze-menu" visibility-while-frozen="withinDistance: 100; requireHoverOnNonMobile: false;" avatar-volume-controls>
```

Comment:
Would be nice if we can ditch `matrix-auto-update` here

Reply:
+1

Reply:
this was here before, i believe due to `billboard` part of `aframe-billboard-component`